code/web/src/modules/user/ProfileDetails.js

```diff
@@ -28,15 +26,20 @@ class ProfileDetails extends Component {
 			image: '',
 			editMode: false,
 			isLoading: false,
+			editPhotoMode: false
 		}
 	}
 
-	componentDidMount = () => {
+
+
+	componentDidMount = async() => {
```

Comment:
I'm curious if we still need the async and await in this? I'm not sure that we do.